src/server.js

```diff
@@ -138,20 +144,30 @@ let load = loader({
       // Create server and start listening
       let server = http.createServer(app);
       await new Promise((accept, reject) => {
-        server.listen(cfg.server.port, accept);
+        server.listen(cfg.server.port);
+        server.once('listening', () => {
+          console.log("Listening on port: " + cfg.server.port);
+        });
         server.once('error', reject);
+        server.once('close', accept);
       });
-      console.log("Listening on port: " + cfg.server.port);
+    },
+  },
+
+  scanner: {
+    requires: ['cfg', 'authorizers'],
+    setup: async ({cfg, authorizers}) => {
+      await scanner(cfg, authorizers);
     },
   },
 }, ['profile']);
 
 if (!module.parent) {
-  load('server', {
+  load(process.argv[3], {
     profile: process.argv[2]
```

Comment:
Note: IMO we should probably make the whole reading from `process.argv` thing a part of `taskcluster-lib-loader`...

Maybe give it a `main` target by default, which does the right thing (tm), I'm getting tired of duplicating the this code...